### PR TITLE
Added HuggingFace links for CCT and Timeseries classification from scratch

### PR DESCRIPTION
**Pull request by EdAbati**
As part of the HuggingFace Keras sprint, I have added the HF links in the CCT and the Timeseries classification from scratch examples.

### CODE REVIEW

**Comment by pcoet** [comment]
@EdAbati Thanks for the PR. It looks like one of the checks failed, and unfortunately the log has expired so I can't investigate. Are you still working on this? Please let us know. Otherwise we'll close the request. Thanks!

**Comment by EdAbati** [comment]
Hi @pcoet , I forgot about this one. :) Happy to have a look at it

**Comment by EdAbati** [comment]
Hi @pcoet , I have updated the branch with master and it looks like this did the trick!

File: examples/timeseries/timeseries_classification_from_scratch.py [hunk 1]
```diff
@@ -224,4 +224,10 @@ def make_model(input_shape):
 after 200 epochs. Beyond the 200th epoch, if we continue on training, the validation
 accuracy will start decreasing while the training accuracy will continue on increasing:
 the model starts overfitting.
+
+Example available on HuggingFace:
```
Comment (pcoet) [inline examples/timeseries/timeseries_classification_from_scratch.py:228]: I think that you need a break between the intro sentence (`Example available on HuggingFace:`) and the table, to make the formatting work. Can you add that and regenerate?

File: examples/vision/cct.py [hunk 2]
```diff
@@ -384,4 +384,9 @@ def run_experiment(model):
 
 The authors also demonstrate the performance of Compact Convolutional Transformers on
 NLP tasks and they report competitive results there.
+
+Example available on HuggingFace:
```
Comment (pcoet) [inline examples/vision/cct.py:388]: See previous comment. I think you need a return between the intro text and the table to make the link table render correctly. That's what I'm seeing in the notebook. Can you add that and regenerate?

**Review by pcoet — CHANGES_REQUESTED** [review]
I'm seeing one formatting issue when I look at these new links in a notebook. Can you fix?

**Review by pcoet — APPROVED** [review]

**Pull request merged**